HW0/P4/P4.py

```diff
@@ -40,37 +44,92 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    measured_inputs = np.loadtxt('P4_measurements.txt', delimiter=',')
+    stretch = np.array([[1.0/rx, 0.0, 0.0],[0.0, 1.0/ry, 0.0],[0.0, 0.0, 1.0/rz]])
+    stretched_inputs = np.dot(stretch, measured_inputs.T)
```

Comment:
You can rewrite this more succinctly by using

stretch.dot(measured_inputs.T)

But what you did is fine!